src/utils.ts

```diff
@@ -32,8 +34,12 @@ export const convertToPrimitiveRepresentation = (type: Type): PrimitiveType => {
       return PrimitiveType.Number;
     case 'boolean':
       return PrimitiveType.Boolean;
+    case 'true':
+      return true;
+    case 'false':
+      return false;
     default:
-      return PrimitiveType.Nothing;
+      return type.getText();
```

Comment:
Since we already have declared `const  text = type.getText()` above, just replace this with `text`.

Reply:
replaced with `text` in last commit